R/tt_to_tblfile.R

```diff
@@ -293,6 +297,7 @@ tt_to_tlgrtf <- function(
     combined_rtf = FALSE,
     one_table = TRUE,
     border_mat = make_header_bordmat(obj = tt),
+    round_type = get_round_type(tt),
```

Comment:
this will incrase version dependency on rtables/rlistings/formatters. Can't forget to update that once those PRs go in, before we merge this one